apps/site-landing/src/containers/App/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,10 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
It's more to test that the page renders than any useful testing of what is rendered.  I think it was disabled for most of our dev life because we were just changing so often